### PR TITLE
updating with application stuff

### PR DESCRIPTION
**Pull request by barnjamin**

### CODE REVIEW

File: pytealutils/applications/application.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+        _impl.__name__ = self.fn.__name__
+        _impl.__annotations__ = self.fn.__annotations__
+        _impl.__signature__ = signature(self.fn)
+        _impl.__doc__ = self.fn.__doc__
+
+        return _impl
+
+
+class AppendExpr:
```
Comment (tzaffi) [inline pytealutils/applications/application.py:66]: Is this used anywhere?

File: pytealutils/applications/application.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    return handler_config
+
+
+def add_handler_config(fn: HandlerFunc | ABIReturnSubroutine, key: str, val: Any):
+    handler_config = get_handler_config(fn)
+    handler_config[key] = val
+    setattr(fn, _handler_config_attr, handler_config)
+
+
+class PrependExpr:
```
Comment (tzaffi) [inline pytealutils/applications/application.py:49]: Is this descriptor magic integral to the goals of the PR? Could similar outcomes be achieved using properties?

File: pytealutils/applications/application.py (hunk 1 above)
Comment (barnjamin) [reply pytealutils/applications/application.py:66]: t'isnt, removed locallyy

File: pytealutils/applications/application.py (hunk 2 above)
Comment (tzaffi) [reply pytealutils/applications/application.py:49]: Sounds like it probably is integral. I'll have a look later in the evening to get better understanding.

**Pull request merged**